index.test.js

```diff
@@ -0,0 +1,54 @@
+import { exportAllDeclaration } from '@babel/types';
+import { TestScheduler } from 'jest';
```

Comment:
remove

Reply:
Thank you reviw！
Removed module loading.

[9c3b694](https://github.com/nyattsu72/handson-jest/pull/1/commits/9c3b6940169e4f10e3d2ee6706db0f3b7e840df9)